CMakeLists.txt

```diff
@@ -138,8 +139,8 @@ macro(get_boost)
 		      set(BOOST_SOURCEDIR "${CMAKE_SOURCE_DIR}/external/boost")       # For internal use
 		      set(BOOST_INSTALLDIR "${CMAKE_BINARY_DIR}/external/boost")      # For internal use
 		      set(BOOST_ROOT "${BOOST_SOURCEDIR}")                            # find_package(Boost ...) hint
-		      set(BOOST_INCLUDEDIR "${BOOST_SOURCEDIR}")                      # find_package(Boost ...) hint
-		      set(BOOST_LIBRARYDIR "${BOOST_INSTALLDIR}/libs")                # find_package(Boost ...) hint
+		      set(BOOST_INCLUDE_DIRS "${BOOST_SOURCEDIR}")                      # find_package(Boost ...) hint
+		      set(BOOST_LIBRARY_DIRS "${BOOST_INSTALLDIR}/lib")                # find_package(Boost ...) hint
```

Comment:
Is there a reason for changing these?
According to [FindBoost](https://cmake.org/cmake/help/latest/module/FindBoost.html), the former are hints while the latter are variable set by find_package(boost) and thus shouldn't be set by the app cmake.

Reply:
Ha yes... i have no see hint value... i have only see result and cache
But second fail is libs folder no exist, is lib

Reply:
fixed.